### PR TITLE
task: update drilldown nodes

### PR DESCRIPTION
**Pull request by blanchco**
# Description

* small change to the drilldown node so that we do not need to define a tab name if there is only 1 top level component

Updated all nodes to use the drilldown component:
https://github.com/DARPA-ASKEM/terarium/issues/2324

### CODE REVIEW

**Comment by blanchco** [comment]
@YohannParis @mwdchang I'm going to update like 3-4 drilldown nodes at a time in separate PRs and merge them all at once so that I don't block any dev work.  Starting with some easy ones here

File: packages/client/hmi-client/src/workflow/ops/code-asset/tera-code-asset-wrapper.vue [hunk 1]
```diff
@@ -1,15 +1,22 @@
 <template>
-	<tera-code :asset-id="props.node?.state?.codeAssetId ?? ''" is-preview />
+	<tera-drilldown :title="node.displayName" @on-close-clicked="emit('close')">
+		<section>
+			<tera-code :asset-id="props.node?.state?.codeAssetId ?? ''" is-preview />
```
Comment (mwdchang) [inline packages/client/hmi-client/src/workflow/ops/code-asset/tera-code-asset-wrapper.vue:4]: Should always have "node" and "node.state", no need for `?` there.

**Comment by mwdchang** [comment]
Didn't test, but code looks fine to me.

**Review by YohannParis — COMMENTED** [review]
It would have been nice to rename those component with the word `drilldown` to understand their purpose, like we do with `node`.

**Pull request merged**